### DISCUSSION TITLE
network error

### DISCUSSION THREAD

**Discussion by manmastersg** [grafana/grafana-infinity-datasource#62] [Q&A]
Hi Sir,

I tried to test it out but I get the following network error
![image](https://user-images.githubusercontent.com/83204957/116040249-5a012480-a69e-11eb-8d81-e67759d5e75e.png)

How can I troubleshoot next?

**Comment by sdvidal** [comment]
Hi, I am hitting same problem...any hints ? thanks!

**Reply by yesoreyeram** [reply]
Can you confirm the data source config doesn't have any url specified and full url is specified only in the query?
